long-ride-alerts/src/test/java/org/apache/flink/training/exercises/longrides/LongRidesUnitTest.java

```diff
@@ -41,6 +43,9 @@ public class LongRidesUnitTest extends LongRidesTestBase {
 
     private KeyedOneInputStreamOperatorTestHarness<Long, TaxiRide, Long> harness;
 
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
```

Comment:
1. What is it? Purpose.
2. Why it's public?

Reply:
Rewritten the Junit to handle exception using Junit-5